go/vt/vttablet/grpctmclient/client.go

```diff
@@ -730,17 +730,22 @@ func (client *Client) SlaveWasRestarted(ctx context.Context, tablet *topodatapb.
 }
 
 // StopReplicationAndGetStatus is part of the tmclient.TabletManagerClient interface.
-func (client *Client) StopReplicationAndGetStatus(ctx context.Context, tablet *topodatapb.Tablet) (*replicationdatapb.Status, error) {
+func (client *Client) StopReplicationAndGetStatus(ctx context.Context, tablet *topodatapb.Tablet, stopIOThreadOnly bool) (hybridStatus *replicationdatapb.Status, status *replicationdatapb.StopSlaveStatus, err error) {
```

Comment:
Any reason for naming the output parameters? IMHO - it always makes me do a double take and look for why this pattern is used. It's sometimes a good tool to use, but I'm not following what the purpose for it is here - you are not even using the names in the method, right?

Reply:
Just my personal opinion: I prefer named outputs, because they clarify what it is that is returned. The variable names are self-documenting.

Reply:
I did this simply for clarity. I think in this case it's confusing to a reader. We are returning two different types of statuses. Why? What are they for? `> 2` returns and I think it warrants named output parameters unless it's already clear what's being returned and why.